Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {
+        Scanner sc = new Scanner(System.in);

```

Comment:
**Suggestion**: Remove the Scanner object instantiation from the "printEvenNumbers" method as it is unused, which cleans up the code and reduces unnecessary object creation. <br> **Code Suggestion**: 
 ```
 -        Scanner sc = new Scanner(System.in);
 ```